.buildkite/dagster-buildkite/dagster_buildkite/utils.py

```diff
@@ -307,6 +307,19 @@ def has_dagster_airlift_changes():
     return any("dagster-airlift" in str(path) for path in ChangedFiles.all)
 
 
+@functools.lru_cache(maxsize=None)
```

Comment:
why no maxsize?

Reply:
Prior art. Basically means that it always uses the first call's result since no params.